docs/spec/Specifiers/Port-Instance-Specifiers.adoc

```diff
@@ -150,6 +152,8 @@ space on the queue for the message.
 
 .. `drop` means that the message is dropped.
 
+.. `hook` means that the message is passed to a user-supplied hook function.
```

Comment:
Add this to the user guide as well https://nasa.github.io/fpp/fpp-users-guide.html#Defining-Components_Port-Instances_Queue-Full-Behavior